### PR TITLE
Fix format

### PR DESCRIPTION
**Pull request by bmarty**
Small changes to generate Kotlin code that please our lint rules.

Could be nice to merge it before the end of the week, since automatic integration to Element Android is set up and it will create a PR with bad code formatting :)

### CODE REVIEW

File: scripts/mobile_generator.py [hunk 1]
```diff
@@ -117,43 +117,48 @@ def compute_kotlin(klass, members, enums, event_name):
     for member in members:
         if member.description:
             result += (
-                f'    /**\n'
-                f'     * {member.description}\n'
-                f'     */\n'
+                f'        /**\n'
+                f'         * {member.description}\n'
+                f'         */\n'
```
Comment (bmarty) [inline scripts/mobile_generator.py:122]: 4 more spaces here

File: scripts/mobile_generator.py [hunk 2]
```diff
@@ -117,43 +117,48 @@ def compute_kotlin(klass, members, enums, event_name):
...
             )
         if member.required:
             defaultValue = ""
         else:
             defaultValue = "? = null"
+        result += "        "
```
Comment (bmarty) [inline scripts/mobile_generator.py:128]: and here (8 instead of 4)

**Review by pixlwave — APPROVED** [review]
LGTM 👍

I experimented with an indent function towards the end of the tweaks I made for Swift. It was something like

```python
def indent(amount):
    return "    " * amount
```

And could then be used as so
```python
result += indent(2) + f'something{var}'
```

Do you think that would make things easier/more readable to handle changes like this in the future?

**Comment by bmarty** [comment]
Thanks for the review.
Yes, it's definitely nicer 👍

**Pull request merged**